src/api/spec/models/bs_request_action/differ/query_builder_for_superseded_spec.rb

```diff
@@ -0,0 +1,195 @@
+require 'rails_helper'
+
+RSpec.describe BsRequestAction::Differ::QueryBuilderForSuperseded do
+  let!(:user) { create(:confirmed_user, login: 'moi') }
+  let!(:source_project) { create(:project, name: 'source_project', maintainer: user) }
+  let!(:source_package) { create(:package, name: 'source_package', project: source_project) }
+  let!(:target_project) { create(:project, name: 'target_project') }
+  let!(:target_package) { create(:package, name: 'target_package', project: target_project) }
```

Comment:
:thought_balloon: you sure like to hit your `!` key ;-)

Reply:
Sure! These objects / factories are only valid all together so it does not make sense to lazy evaluate them.